src/main/scala/io/sdkman/changelogs/java/BellSoftLibericaMigrations.scala

```diff
@@ -565,18 +142,18 @@ class BellSoftLibericaMigrations {
         version =>
           removeVersion(
             "java",
-            "115.0.1-librca",
+            "8.0.272,fx-librca",
```

Comment:
```suggestion
            "8.0.272.fx-librca",
```

Reply:
Thx! I fixed it now.
commit is https://github.com/sdkman/sdkman-db-migrations/pull/408/commits/85ea3e324eaf9a9de2be47e344a88a915b59041c